include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -115,21 +116,22 @@
 #endif //Enable SIMD
 
 // Enable loop unrolling pragmas where supported
-#if (__INTEL_COMPILER || (!defined(__INTEL_COMPILER) && _ONEDPL_GCC_VERSION >= 80000))
+#if (__INTEL_LLVM_COMPILER || __INTEL_COMPILER ||                                                                      \
+    (!defined(__INTEL_LLVM_COMPILER) && !defined(__INTEL_COMPILER) && _ONEDPL_GCC_VERSION >= 80000))
 #    define _ONEDPL_PRAGMA_UNROLL _ONEDPL_PRAGMA(unroll)
 #else //no pragma unroll
 #    define _ONEDPL_PRAGMA_UNROLL
 #endif
 
-#if (__INTEL_COMPILER)
+#if (__INTEL_LLVM_COMPILER || __INTEL_COMPILER)
 #    define _ONEDPL_PRAGMA_FORCEINLINE _ONEDPL_PRAGMA(forceinline)
 #elif defined(_PSTL_PRAGMA_FORCEINLINE)
 #    define _ONEDPL_PRAGMA_FORCEINLINE _PSTL_PRAGMA_FORCEINLINE
 #else
 #    define _ONEDPL_PRAGMA_FORCEINLINE
 #endif
 
-#if (__INTEL_COMPILER >= 1900)
+#if (__INTEL_LLVM_COMPILER >= 20230100 || __INTEL_COMPILER >= 1900)
```

Comment:
What' about compilers `__INTEL_LLVM_COMPILER  < 20230100` ?
Are  these covered by `__INTEL_COMPILER >= 1900` condition?

Reply:
you want to say that DPC++ witch __INTEL_LLVM_COMPILER < 20230100, doesnt support features defined on lines 135.....

Reply:
I that case, I think, it makes sense to add a description in the code... otherwise, such condition looks as non quite correct  - we "skip"  the compliers which `__INTEL_LLVM_COMPILER < 20230100.`..

Reply:
> I that case, I think, it makes sense to add a description in the code... otherwise, such condition looks as non quite correct condition - we "skip" the compliers which __INTEL_LLVM_COMPILER < 20230100...

@MikeDvorskiy, I guess this does not look straightforward probably because of an assumption that DPCPP is a linear continuation of ICC, so let's say DPCPP 2021 should support features which are supported by ICC 2019. This does not look right when comparing e.g. [OpenMP* Features and Extensions Supported in Intel® oneAPI DPC++/C++ Compiler](https://www.intel.com/content/www/us/en/developer/articles/technical/openmp-features-and-extensions-supported-in-icx.html) and [A Survey of OpenMP* Features Implemented in Intel® Fortran and C++ Compilers](https://www.intel.com/content/www/us/en/developer/articles/technical/a-survey-of-openmp-features-implemented-in-intel-fortran-and-c-compilers.html).

Does it solve your confusion (if I guessed it right)? Would you still prefer adding an explanation?